src/prisma/schema.prisma

```diff
@@ -3678,3 +3679,41 @@ model ScoutProjectWalletPollEvent {
   @@index([walletId])
   @@index([toBlockNumber])
 }
+
+enum ScoutNotificationChannel {
+  farcaster
+  email
+}
+
+model ScoutNotification {
```

Comment:
move sentAt, userId and notificationType to individual tables and get rid of this one